distribution/docs/src/main/jdocs/content/_plugins/geocoder-plugin-contents.adoc

```diff
@@ -0,0 +1,33 @@
+:type: plugin
+:status: published
+:title: GeoCoder Plugin
+:link: _geocoder_plugin
+:plugintypes: preingest
+:summary: Populates the `Location.COUNTRY_CODE` attribute if the Metacard has an associated location.
+
+The GeoCoder Plugin is a pre-ingest plugin that is responsible for populating the Metacard's `Location.COUNTRY_CODE` attribute if the Metacard has an associated location.
```

Comment:
✏️  we seem to be inconsistent in how we spell `Pre-Ingest` or `pre-ingest`. Should we come up with a common spelling for it in our docs?